homeassistant/components/flipr/__init__.py

```diff
@@ -1,22 +1,60 @@
 """The Flipr integration."""
 
+from collections import Counter
+from dataclasses import dataclass
+import logging
+
+from flipr_api import FliprAPIRestClient
+
 from homeassistant.config_entries import ConfigEntry
-from homeassistant.const import Platform
+from homeassistant.const import CONF_EMAIL, CONF_PASSWORD, Platform
 from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryError
+from homeassistant.helpers import issue_registry as ir
 
 from .const import DOMAIN
 from .coordinator import FliprDataUpdateCoordinator
 
 PLATFORMS = [Platform.BINARY_SENSOR, Platform.SENSOR]
 
+_LOGGER = logging.getLogger(__name__)
+
+
+@dataclass
+class FliprData:
+    """The Flipr data class."""
+
+    flipr_coordinators: list[FliprDataUpdateCoordinator]
```

Comment:
If you only store the coordinators, no need to wrap it in a dataclass, so you can type ConfigEntry to contain the list of coordinators by default

Reply:
In hub, I plan to use hub_coordinators as a second list in the data class.

Reply:
wdym in hub?

Reply:
Sorry, when implementing the hub, the dataclass will become : 
```
@dataclass
class FliprData:
    flipr_coordinators: list[FliprDataUpdateCoordinator]
    hub_coordinators: list[HubDataUpdateCoordinator]
```
since the hub coordinators use a different api method to retrieve data